conf.c

```diff
@@ -309,6 +309,21 @@ conf_set_type(char *str)
 	return (-1);
 }
 
+int
+conf_validate_state(struct bsm_state *bm)
```

Comment:
Seems like it wouldn't be unreasonable to move this function into grammar.y entirely, since it's a small validation thing.